Framework/Kernel/src/SplittingInterval.cpp

```diff
@@ -55,6 +55,13 @@ bool SplittingInterval::operator==(const SplittingInterval &ti) const {
   }
 }
 
+std::string SplittingInterval::debugStrPrint() const {
```

Comment:
Is it intended to keep this function in the code in case we need it in the future?

Reply:
yes